terraform/modules/networking/subnet.tf

```diff
@@ -0,0 +1,49 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+locals {
+  nat_gateway_cidr_10 = cidrsubnet(aws_vpc.main.cidr_block, 6, 0) # 0.0/22 - public
```

Comment:
what does the 10 mean here?

Reply:
It's the number of bits available for addresses inside the subnet (convention copied from Delta - I'm not in any way wedded to it!)